jupyterhub/tests/test_singleuser.py

```diff
@@ -202,6 +212,9 @@ def assert_not_in_home(path, name):
 async def test_notebook_dir(
     request, app, tmpdir, user, full_spawn, extension, notebook_dir
 ):
+    if IS_JUPYVERSE:
+        pytest.skip("jupyter-server specific test")
```

Comment:
This test is not jupyter-server specific, it's verifying if the root directory config is respected. The mechanism of the test (specifically the use of the test extension utility) may need different handling, but the API calls to the contents API should produce the same results.

Reply:
Jupyverse has no root directory config (yet), what do you think about re-enabling this test if/when there is support for it?

Reply:
@minrk what do you think about merging this?